### PR TITLE
Feature/send msg to collector

### PR DESCRIPTION
**Pull request by salceson**
@mariusz89016, have you forgotten about creating this pull request?

### CODE REVIEW

**Comment by mariusz89016** [comment]
Ok, I've tried it and everything works fine. 
If you can @salceson , please check comments.

File: build.sbt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name := "distr_stacktrace"
 
-version := "1.0"
+organization := "org.mariusz89016"
```
Comment (mariusz89016) [inline build.sbt:3]: It was first the first thing that came to my mind :D
Maybe:

```
organization := "pl.edu.agh.iet"
name := "akka-debugging-tool"
```

?

File: src/main/scala/akka_debugging/collector/Collector.scala [hunk 2]
```diff
@@ -32,41 +25,29 @@ trait Collector extends Actor {
...
-    case CollectorExceptionMessage(actorRef: ActorRef, uuid: UUID, exception: Exception) =>
-      val niceStackTrace = filterStackTrace(exception.getStackTrace)
-      fileWriter.write(String.format("===========\nEXCEPTION\n===========\n%s\n%s\n%s\n%s\n",
-        actorRef.toString, uuid.toString, exception.getClass.getCanonicalName, niceStackTrace.mkString("\n")))
-      fileWriter.flush()
+    case msg: RelationMessage =>
+      handleRelationMessage(msg)
   }
 }
+//class FileCollector extends Collector {
```
Comment (mariusz89016) [inline src/main/scala/akka_debugging/collector/Collector.scala:29]: Uncomment it & check if it works.

File: src/main/scala/akka_debugging/database/DatabaseUtils.scala [hunk 3]
```diff
@@ -1,26 +1,16 @@
...
-                              stackTrace: String,
-                              id: Option[Long] = None)
-
-
-case class CollectorDBExceptionMessage(uuid: UUID,
-                                       actor: String,
-                                       exception: String,
-                                       stackTrace: String,
-                                       id: Option[Long] = None)
+case class CollectorDBMessage(id: Int,
```
Comment (mariusz89016) [inline src/main/scala/akka_debugging/database/DatabaseUtils.scala:9]: For now it's only simplified version. I think we can leave it for now and change it in appropriate way in the future.

File: src/main/scala/akka_debugging/database/DatabaseUtils.scala [hunk 4]
```diff
@@ -34,37 +24,17 @@ object DatabaseUtils {
...
-    def message = column[String]("message")
-
-    def stackTrace = column[String]("stackTrace")
-
-    override def * = (uuid, actor, message, stackTrace, id.?) <>(CollectorDBMessage.tupled, CollectorDBMessage.unapply)
-
-    def uuidIndex = index("messages_uuid_idx", uuid, unique = true)
+    def id = column[Int]("id", O.PrimaryKey)
+    def sender = column[String]("sender")
+    def receiver = column[String]("receiver", O.Nullable) // -.- postgres (deprecated)
```
Comment (mariusz89016) [inline src/main/scala/akka_debugging/database/DatabaseUtils.scala]: I had problem with that - could you check if I made this in a proper way?

**Comment by mariusz89016** [comment]
We also have to think about:
- packages names
- removing unnecessary code (comments)
- optimize imports
- how we want to handle stacktrace

File: build.sbt (hunk 1 above)
Comment (salceson) [inline build.sbt:3]: Don't worry about this now, it'll be corrected during my task. But yes, these are good remarks.

File: src/main/scala/akka_debugging/collector/Collector.scala (hunk 2 above)
Comment (salceson) [inline src/main/scala/akka_debugging/collector/Collector.scala:29]: It doesn't handle one of the message passing (about correlation), I think.

File: src/main/scala/akka_debugging/database/DatabaseUtils.scala (hunk 3 above)
Comment (salceson) [inline src/main/scala/akka_debugging/database/DatabaseUtils.scala:9]: Yeah, they'll be modified.

File: src/main/scala/akka_debugging/database/DatabaseUtils.scala (hunk 4 above)
Comment (salceson) [inline src/main/scala/akka_debugging/database/DatabaseUtils.scala]: Hmmm - exactly what are you referring to?

**Comment by salceson** [comment]
@mariusz89016 if you still have some comments, please write them here, I'm merging this.

**Pull request merged**